src/tutorials/0004-mutable-file-system/05.md

```diff
@@ -23,12 +23,11 @@ such as `/catPics`,
 `files.ls` produces an array of objects, one for each file or directory
 contained in the directory you're inspecting, with the following properties:
 
+- `cid`: the Content Identifier (CID) that identifies your file in IPFS
 - `name`: the file's name
-- `type`: the object's type (`0` - file or `1` - directory)
+- `type`: the object's type (`file` or `directory`)
 - `size`: the size of the file in bytes
-- `cid`: the Content Identifier (CID) that identifies your file in IPFS
 - `mode`: the UnixFS mode as a Number
-- `mtime`: an object with numeric `secs` and `nsecs` properties
```

Comment:
This field will be present if found in the DAG so it's probably worth keeping it here with a note that it's an optional property.

That is, `mode` gets a [default value](https://github.com/ipfs/js-ipfs-unixfs/blob/master/packages/ipfs-unixfs/src/index.js#L24-L25) depending on if the entry is a file or directory, but `mtime` does not.